src/pilot/ironic.patch

```diff
@@ -1,6 +1,99 @@
---- ironic.conf	2020-08-07 13:16:47.000000000 -0400
-+++ ironic.conf.new	2020-09-08 15:21:43.975308033 -0400
-@@ -1291,7 +1291,7 @@
+--- ironic.conf	2020-09-17 02:13:45.000000000 +0000
++++ ironic.conf.new	2020-09-18 16:37:38.844775481 +0000
+@@ -37,7 +37,7 @@
+ # enumerating the "ironic.hardware.types" entrypoint. (list
+ # value)
+ #enabled_hardware_types = ipmi
+-enabled_hardware_types=idrac,ilo,ipmi,redfish
++enabled_hardware_types=idrac
```

Comment:
Is there a reason for shutting off the ilo, ipmi, and redfish hardware types?  In the enabled_*_interfaces below, ilo and redfish are enabled, which seems odd when the hardware type is disabled.

Reply:
Not enabling the other drivers enables us to assign hardware interface implementation values to the `default_xxx_interface` settings which are not supported by those drivers. The Conductor ensures all `default_xxx_interface`s are supported by all enabled drivers. It fails to start if it finds they are not.

We could remove the `default_xxx_interface` settings. However, that may make it more challenging to switch from WS-Man to Redfish interface implementations.

Reply:
Ah, that makes sense. Thanks!